nearuplib/nodelib.py

```diff
@@ -18,27 +20,75 @@
     download_genesis,
     latest_genesis_md5sum,
     new_release_ready,
+    prompt_flag,
 )
 from nearuplib.watcher import is_watcher_running, run_watcher, stop_watcher
 from nearuplib.tailer import next_logname
 
 
-def init_near(home_dir, binary_path, chain_id, init_flags):
+def read_validator_key(home_dir):
+    key_path = os.path.join(home_dir, 'validator_key.json')
+    if not os.path.exists(key_path):
+        return None
+
+    with open(key_path) as key_file:
+        return json.loads(key_file.read())
+
+
+def print_validator_info(home_dir):
+    key_data = read_validator_key(home_dir)
+    if key_data is None:
+        logging.warning('Validator key not generated by neard as expected')
+        return
+
+    print('The validator\'s public key is {}'.format(key_data['public_key']))
+    print(textwrap.fill(textwrap.dedent("""\
+        The node will now run and will sync with the network, but this
+        will not automatically initiate staking. In order to do that,
+        you need to send a transaction to the network indicating your
+        intent to stake. Please see
+        https://docs.near.org/docs/develop/node/validator/staking-and-delegation.
+
+        If you have near-cli (https://github.com/near/near-cli) installed, you
+        can do this with the following command:
+"""), width=80, break_on_hyphens=False, break_long_words=False))
```

Comment:
Perhaps indent so it doesn’t break the flow of the function. textwrap should handle the extra spaces.